client/app/l10n/en.json

```diff
@@ -550,18 +550,18 @@
   "Security awareness panel title": "Security awareness panel title",
   "Select all": "Select all",
   "Select all recipients by default": "Select all recipients by default",
-  "Select an option:": "Select an option:",
+  "Select an option:": "Select something:",
```

Comment:
We considered to keep the current text.

Reply:
How about "Make a selection". Select and option can be the same word in some languages, making the translation quirky.